docker-compose.yml

```
@@ -21,6 +21,7 @@ services:
       - AWS_SECRET_ACCESS_KEY=${AWS_SECRET_ACCESS_KEY:-test}
       - AWS_ENDPOINT_OVERRIDE=http://localstack:4566/
       - AWS_S3_BUCKET_FAILED_FILES=data-store-failed-files-dev
+      - FIND_SERVICE_BASE_URL="http://localhost:4002"
```

```suggestion
      - FIND_SERVICE_BASE_URL=http://localhost:4002
```

Actually, we need to remove the quotes here - docker compose doesn't understand them in its env blocks

---

Quotation marks removed.